src/Classes/Utils/Utils.ts

```diff
@@ -27,6 +27,17 @@ export class Utils {
         } : null;
     }
 
+    /**
+     * Generate a random RGBA color
+     */
+    public static getRandomColorRGBA(min?: number) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'getRandomColorRGBA' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)